packages/site/src/constants/index.ts

```diff
@@ -0,0 +1,35 @@
+import { LinkCardProps } from "@framework/system/src/components/landing/link-card"
+
+export const FRAMEWORK_RESOURCES = ({
+	bigSize,
+}: Pick<LinkCardProps, "bigSize">): LinkCardProps[] => [
```

Comment:
Wouldn't this just be better if we did:
```suggestion
export const FRAMEWORK_RESOURCES = (bigSize: boolean = false): LinkCardProps[] => [
```